paparazzi/paparazzi/src/main/java/app/cash/paparazzi/HtmlReportWriter.kt

```diff
@@ -290,5 +290,5 @@ internal val filenameSafeChars = CharMatcher.inRange('a', 'z')
   .or(CharMatcher.anyOf("_-.~@^()[]{}:;,"))
 
 internal fun String.sanitizeForFilename(): String? {
-  return filenameSafeChars.negate().replaceFrom(toLowerCase(Locale.US), '_')
+  return filenameSafeChars.negate().replaceFrom(lowercase(Locale.US), '_')
```

Comment:
Ah, ok, looking at https://cashapp.github.io/paparazzi/changelog/#new Paparazzi is just using the latest Kotlin. `toLowerCase` is `@DeprecatedSinceKotlin(warningSince = "1.5")` and `lowercase` is `@SinceKotlin("1.5")`, so that warning must've been around a while.

Note that there was a huge jump in Kotlin with Gradle 8 upgrade:
https://docs.gradle.org/current/userguide/compatibility.html#kotlin
so I think we have to be careful using Kotlin 1.5-1.8 functions in the Gradle Plugin (only) **iff** Paparazzi wants to keep supporting AGP 7.x.

@jrodbx are there any compatibility / supported version ranges doc? like [Detekt](https://detekt.dev/docs/next/introduction/compatibility).

Reply:
@TWiStErRob compatibility, at the moment, is whatever minor version of AGP is used in Paparazzi.